README.md

```diff
@@ -39,7 +39,29 @@ Please file an issue if additional packages are required.
 * python3.9
 * python3.9-venv
 * musl-1.2.2
-* ARM GCC compiler; 10.2-2020.11
+* cmake
+* ninja-build
+* ARM GCC compiler for none-elf; 10.2-2020.11
+* aarch64 Linux gcc compiler
```

Comment:
This is not necessary anymore either. `aarch64-none-elf-` is used for everything.

Reply:
It breaks without it.

Reply:
Can you show me the error message? The dependencies installed for the CI are here https://github.com/seL4/microkit/blob/89d1e621da862a0cbe33e0ac333eae61b579286c/.github/workflows/sdk.yaml#L33 and no longer use `aarch64-linux-gnu-`. The use of it was removed in https://github.com/seL4/microkit/commit/cb849f7695a39d3e0a9e2b34f7c9e360992a8883.

Reply:
This may have been because I was testing on a different branch.  Repeating on master now